docs/apple_development.md

```diff
@@ -3,7 +3,7 @@
 This document describes how to develop and debug for macOS (formerly known as OS X) and iOS. You will need:
 
 - An Apple Developer Account. You will need to be invited to your developer team as well.
-- XCode 13.2 ([download](https://developer.apple.com/xcode/))
+- XCode ([download](https://developer.apple.com/xcode/))
```

Comment:
Could you add the version we support here. It was there because we didn't support other versions.

Do you mean we support any XCode version now?

Reply:
Yeah, if we build on setup, we'll be able to support any xcode verison

Reply:
This makes it easier for random people to contribute

Reply:
I think it's still valuable to mention the min version we support, which is used in our CI: https://github.com/Jigsaw-Code/outline-client/blob/bd1fc961365fa4920107bbb90a6af7233ad8ac05/.github/workflows/build_and_test_debug.yml#L107
```suggestion
- XCode 13.2+ ([download](https://developer.apple.com/xcode/))
```